framework/src/org/apache/cordova/engine/SystemWebViewClient.java

```diff
@@ -116,6 +118,18 @@ public SystemWebViewClient(SystemWebViewEngine parentEngine) {
         });
 
         this.assetLoader = assetLoaderBuilder.build();
+        boolean setAsServiceWorkerClient = parentEngine.preferences.getBoolean("ResolveServiceWorkerRequests", false);
```

Comment:
I feel like if this is going to be introduced in the 13.x release then we could default to `true` so that service workers just "works" out of the box.

But I don't have a strong opinion to block this and I'm okay if this remains using `false` as a default.

Reply:
I've set it to be the same default value as `cordova-plugin-ionic-webview` in case people is migrating from there, but if we want it to be `true` and it's not going to be released as a minor, I can change it.